fe/fe-core/src/main/java/com/starrocks/connector/iceberg/rest/IcebergRESTCatalog.java

```diff
@@ -59,59 +61,72 @@
 import static com.starrocks.connector.iceberg.IcebergApiConverter.convertDbNameToNamespace;
 import static com.starrocks.connector.iceberg.IcebergCatalogProperties.ICEBERG_CUSTOM_PROPERTIES_PREFIX;
 import static com.starrocks.connector.iceberg.IcebergMetadata.LOCATION_PROPERTY;
+import static java.lang.String.format;
+import static org.apache.iceberg.CatalogUtil.configureHadoopConf;
 
 public class IcebergRESTCatalog implements IcebergCatalog {
 
-    private static final Logger LOG = LogManager.getLogger(IcebergRESTCatalog.class);
+    public enum Security {
+        NONE,
+        OAUTH2,
+        OIDC,
```

Comment:
I think OIDC is an extension of OAuth2， we should not use OIDC as a new  security type,  this security type we can refer to AuthProperties.java  in iceberg 
![image](https://github.com/user-attachments/assets/afbc4bfd-1d8d-4c7d-9df0-1528408a3706)